docs/_docs/permalinks.md

```diff
@@ -115,18 +142,47 @@ Here's the full list of placeholders available:
         <p><code>y_day</code></p>
       </td>
       <td>
-        <p>Day of the year from the post's filename, with leading zeros.</p>
+        <p>Ordinal day of the year from the post’s filename, with leading zeros. (001..366)</p>
       </td>
     </tr>
     <tr>
       <td>
-        <p><code>short_year</code></p>
+        <p><code>w_year</code></p>
       </td>
       <td>
-        <p>
-          Year without the century from the post's filename. May be overridden
-          via the document’s <code>date</code> front matter
-        </p>
+        <p>Week year which may differ from the month year for up to three days at the start of Janaury and end of December</p>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <p><code>week</code></p>
+      </td>
+      <td>
+        <p>Week number of the current year, starting with the first week having a majority of its days in January. (01..53)</p>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <p><code>w_day</code></p>
+      </td>
+      <td>
+        <p>Day of the week, starting Monday. (1..7)</p>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <p><code>short_day</code></p>
+      </td>
+      <td>
+        <p>Three-letter weekday abbreviation, e.g. _SUN_.</p>
```

Comment:
Similar to the comments above, you'll want an `<i>` tag if this is meant to be italics.
```suggestion
        <p>Three-letter weekday abbreviation, e.g. <i>SUN</i>.</p>
```